README.md

```diff
@@ -0,0 +1,50 @@
+# ⚡️ FuelLabs Design System
+[![License](https://img.shields.io/github/license/FuelLabs/design-system)](https://github.com/FuelLabs/design-system)
+[![Issues Open](https://img.shields.io/github/issues/FuelLabs/design-system)](https://github.com/FuelLabs/design-system)
+[![Github Forks](https://img.shields.io/github/forks/FuelLabs/design-system)](https://github.com/FuelLabs/design-system)
+[![Github Stars](https://img.shields.io/github/stars/FuelLabs/design-system)](https://github.com/FuelLabs/design-system)
+
+Welcome to the source code repository for FuelLabs Design System.
+
+## Getting Started
+
+This project utilizes Storybook for component development. If you would like to know more about Storybook or how it works, [check out their website](https://storybook.js.org/).
+
+### Project Installation
+
+1. Make sure you have the latest stable version of Node. Check out [Configuring Node and NVM to make sure](#configuring-node-and-nvm)
+2. Clone the repository down locally.
+3. `cd` into the project and run `pnpm install` to install all project dependencies.
+4. Run `pnpm build` to build the peer dependencies
+
+### Running the Development Environment
+
+1. `cd` into `packages/react`
+2. Run `pnpm storybook`
+
+Whenever you add, remove, or alter a component's css annotation metadata, you'll need to restart Storybook to see those changes.
```

Comment:
This is not mandatory, you don't need to restart Storybook for every change, it has a cool fast refresh. Do you need this just if change Storybook bundler configurations (living inside `.storybook` folder)